src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let result = 0;
+
+  for (let i = 0; i < days; i++) {
+    result += cost;
+  }
```

Comment:
loop here is redundant, you can simply multiply cost * days or 40 * days

Reply:
thank you